topbeat/docs/fields.asciidoc

```diff
@@ -168,12 +167,12 @@ type: int
 
 The amount of CPU time spent in involuntary wait by the virtual CPU while the hypervisor was servicing another processor. Available only on Unix.
 
-[float]
+
 === cpus Fields
 
 This group contains CPU usage per core statistics.
 
-[float]
```

Comment:
@ruflin Yes, it is required. It's a manual hack because otherwise the doc build breajs the content out into separate topics, which isn't ideal. I think the hack is better than leaving it as-is. But we do need to figure out if we can automate this at some point.